src/refresh.cc

```diff
@@ -579,13 +579,13 @@ refreshCheckHTTP(const StoreEntry * entry, HttpRequest * request)
 }
 
 /// \see int refreshCheckHTTP(const StoreEntry * entry, HttpRequest * request)
-int
-refreshCheckICP(const StoreEntry * entry, HttpRequest * request)
+bool
+refreshCheckICP(const StoreEntry *entry, const HttpRequestPointer &request)
 {
-    int reason = refreshCheck(entry, request, 30);
+    int reason = refreshCheck(entry, request.getRaw(), 30);
     ++ refreshCounts[rcICP].total;
     ++ refreshCounts[rcICP].status[reason];
-    return (reason < 200) ? 0 : 1;
+    return !(reason < 200);
```

Comment:
If you are going to make this condition look different from its duplicates (in refresh.cc context), then please simplify it:

```suggestion
    return reason >= 200;
```

Alternatively, keep the original look-and-feel (until all such conditions are fixed in refresh.cc):

```suggestion
    return (reason < 200) ? false : true;
```